krator/Cargo.toml

```diff
@@ -19,7 +19,7 @@ maintenance = { status = "actively-developed" }
 
 [features]
 default = ["kube-native-tls"]
-kube-native-tls = ["kube/native-tls"]
+kube-native-tls = ["kube/openssl-tls"]
```

Comment:
I don't think the `native-tls` feature should use `openssl`, could you add a different feature?

Reply:
Ah, yes. You are right!